mininterface/tk_interface/utils.py

```diff
@@ -150,7 +150,10 @@ def _fetch(variable):
                 case SecretTag():
                     grid_info = widget.grid_info()
                     widget.grid_forget()
-                    widget = SecretEntryWrapper(master, tag, variable, grid_info).entry
+                    # Create wrapper and store it in the widget list
+                    wrapper = SecretEntryWrapper(master, tag, variable, grid_info)
+                    widget = wrapper.entry
+                    subwidgets.append(wrapper)
```

Comment:
I think it was better before.
Subwidgets seems to be used only for real entries, not wrappers. Or what this should do? :)